### DISCUSSION TITLE
New Update Resolution Settings

### DISCUSSION THREAD

**Discussion by wolf0015** [lllyasviel/Fooocus#3058] [Q&A] [answered]
I've been using Fooocus for the past month and i set my custom resolution settings, but with the new update i cant find where i can either add or modify these settings, can someone tell me what is the new location of them? i knew it was in the config file inside the modules folder before the update. Also, i dont use GitHub very often and i cant find any recent discussions regarding this issue, any help i will be most thankful for. Thank you in advance

**Comment by Rogala** [comment]
[Nothing has changed, all the settings are here](https://github.com/lllyasviel/Fooocus#customization)

**Reply by wolf0015** [reply]
my config.txt file on the Fooocus folder doesnt show anything besides some file locations, everything shown in the config_modification_tutorial.txt file is not on the main config file, what should i do? delete the config.txt file like the README says? i also checked twice the config file in the modules folder and nothing, i cant find the settings anywhere

**Reply by Rogala** [reply, marked as answer]
**config_modification_tutorial.txt** is an example of what you can customize in the main **config.txt file**, transfer the settings you need from **config_modification_tutorial.txt--->config.txt**

**Reply by wolf0015** [reply]
I love you already haha, thank you very much, that was it
